mitodl/micromasters
# Renamed `User*` components to `Learner*`

#### What are the relevant tickets?

closes #2526 

#### What's this PR do?

Renames all of the `User`-prefixed components to `Learner`. So `UserPage -> LearnerPage`, etc. This is just a clarify change, since we always talk about the page which shows info about individual learners as being the 'learner' page.

#### How should this be manually tested?

This is just a rename, there shouldn't be any functional changes. I would just make sure that everything on the `/learner` and `/learners` page is working as it should, and that I didn't miss anything that should be renamed!

Probably a good idea to quickly click around the rest of the app and confirm that things look correct as well.
